packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -536,11 +536,16 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         shouldHighlight: shouldHighlight,
       });
       const barHeight: number = Math.max(yBarScale(point.y), 0);
-      if (barHeight < 1) {
+      let adjustedBarHeight = 0;
```

Comment:
Thanks for fixing this Shubhabrata.

Could you also apply the same fix to the string bars below.

Reply:
Sure... Just tell me if any other PR is to be created for this or should I include it in this PR itself?

Reply:
You can make all the changes in the same PR.